### DISCUSSION TITLE
[response][INFO] (web_files) GET /<p..> [10] => 404 Not Found

### DISCUSSION THREAD

**Discussion by scgf** [dani-garcia/vaultwarden#3733] [Q&A] [answered]
When I look at the docker log file I see this repeated over and over. I am running the latest Vaultwarden container (1.29.1) and the Bitwarden app version 2023.7.0 (9969).

Any idea what it means? Everything works fine but I'm interested to know what it's all about. I'm guessing it might me the website icons in Bitwarden . . .

**Comment by BlackDex** [comment]
If you have a reverse proxy, check those logs to see which files it tries to load. Or check the F12 browser developer tools and see which files it fails to download.

**Reply by scgf** [reply]
I'm running Vaultwarden on a Synology NAS and I do use a reverse proxy, as I do with many other services. I can't find a way of accessing a log for the reverse proxy on my NAS. I'm also unsure about using the F12 browser developer tools to see which files it fails to download. I'm using Librewolf on an M1 mac and when I press f12 I get a developer window but nothing happens in that window when I get Bitwarden to fill a password or sync its database. I just thought it might be a known issue.

**Reply by BlackDex** [reply]
Well, it isn't an issue. It happens when a client (or something else) tries to access files which do not exists.

**Reply by scgf** [reply, marked as answer]
For the benefit of others getting this error - I needed to enable websockets in the Synology reverse proxy. Support isn't enabled by default. You click on Custom Header, Create and Websocket is an available option, already filled in for you.

**Reply by CarnivoreD** [reply]
Does this have any security impact?

**Comment by RheaAyase** [comment]
I'm getting this error all of a sudden, after a long time of running just fine. It's not necessarily a websocket issue, that's fine on my end.

**Reply by RheaAyase** [reply]
- it's 2025.12 update that broke the extension...

**Reply by Solanum95** [reply]
Yeah same issue after 2025.12..

**Reply by RheaAyase** [reply]
It's funny that the patch notes for it don't mention anything that could be a breaking change.

I reverted it until.. a) bitwarden fixes their shit or..  b) i feel like updating my server (assuming vaultwarden will catch up)

**Reply by scgf** [reply]
I moved to bitwarden lite. It's working really well and has recently come out of beta.

**Reply by m4teh** [reply]
Yep same issue. Changes to entries are erroring out (despite still actually saving) and it's frustrating me.

**Reply by Gerardv514** [reply]
> I moved to bitwarden lite. It's working really well and has recently come out of beta.

What are you running it on?

**Reply by scgf** [reply]
> What are you running it on?

I'm running it on my Synology DS723+

**Reply by Gerardv514** [reply]
> 
> 
> > What are you running it on?
> 
> 
> 
> I'm running it on my Synology DS723+

Possible to provide some instructions on how you got that going? I'd be interested in checking it out.

**Reply by scgf** [reply]
No problem. I used the tutorial from [Marius Hosting](https://mariushosting.com/how-to-install-bitwarden-on-your-synology-nas/)

Just bear in mind it is recently out of beta, so the image name is now ghcr.io/bitwarden/lite - you’ll need to make that change in the docker compose file.

There are some notes in the [official Bitwarden pages](https://bitwarden.com/help/install-and-deploy-lite/) too.

**Reply by Overlord7064** [reply]
I am also seeing this error also all of a sudden.  Never seen it before.  Changes from the Windows App get saved even throwing "An Unexpected Error Has occurred" along with "[response][INFO] (web_files) GET /<p..> [10] => 404 Not Found" in the log.  This error is not happening on IOS devices.  Gotta be the new App from Bitwarden 2025.12.0.

**Reply by BlackDex** [reply]
There is a new endpoint which the clients since version 2025.12.0 are using, this endpoint is not available in the current `latest` (a.k.a. stable) version of Vaultwarden. It however is in the current `testing` (a.k.a. beta) version of Vaultwarden container images.

**Comment by Overlord7064** [comment]
Thanks for this. On Dec 21, 2025, at 12:38 PM, Mathijs van Veluw ***@***.***> wrote:﻿
There is a new endpoint which the clients since version 2025.12.0 are using, this endpoint is not available in the current latest (a.k.a. stable) version of Vaultwarden. It however is in the current testing (a.k.a. beta) version of Vaultwarden container images.
